sale_order_priority/models/stock_picking.py

```diff
@@ -0,0 +1,11 @@
+from odoo import models
+
+
+class Picking(models.Model):
+    _inherit = "stock.picking"
+
+    def action_assign(self):
+        res = super().action_assign()
+        picking_priority = self.env.context.get("sale_priority")
+        self.priority = picking_priority if picking_priority else 0
```

Comment:
```suggestion
    def action_confirm(self):
        res = super().action_confirm()
        picking_priority = self.env.context.get("sale_priority", False)
        self.write({"priority": picking_priority or 0})
```

Reply:
From what I've tested, the action_assign method doesn't seem to do the trick anymore. With the action_assing the module was not working, the priority of the order was not passed to the picking. Applying the changes,  it should work again! :+1:

Reply:
@GuillemCForgeFlow Thanks! it's working now